README.md

```diff
@@ -1,8 +1,36 @@
-# Serverless Framework: Deploy on Scaleway Functions
+# Scaleway Plugin for Serverless Framework
 
-The Scaleway functions plugin for [Serverless Framework](https://serverless.com/) allows users to deploy their functions and containers to Scaleway Functions with a simple `serverless deploy`.
+This is the [Scaleway Functions](https://www.scaleway.com/en/serverless-functions/) plugin for [Serverless Framework](https://serverless.com/).
 
-Serverless Framework handles everything from creating namespaces to function/code deployment by calling APIs endpoint under the hood.	
+## Quick-start
+
+0. Install the [Serverless Framework](https://www.serverless.com/framework/docs/getting-started).
+1. Create a Scaleway account, and log in to the [Scaleway Console](https://console.scaleway.com/login).
+2. Generate an [API key](https://www.scaleway.com/en/docs/console/my-project/how-to/generate-api-key/).
+3. Set up the [Scaleway CLI](https://www.scaleway.com/en/cli/).
+4. Generate and invoke your function:
+
+```shell
+# Available runtimes: https://github.com/scaleway/serverless-scaleway-functions/tree/master/examples
```

Comment:
List of available runtimes is available here : https://www.scaleway.com/en/docs/compute/functions/reference-content/functions-lifecycle/#available-runtimes

Reply:
Yes, the variable is poorly named perhaps, as it's actually used in: `//github.com/scaleway/serverless-scaleway-functions/tree/master/examples/${RUNTIME}`. It's probably better as `EXAMPLE`.